src/pretty/entry.rs

```diff
@@ -0,0 +1,117 @@
+use pretty::BoxDoc;
+use typst_syntax::{ast::*, SyntaxKind};
+
+use super::{comment::comment, util::is_comment_node, PrettyPrinter};
+
+// The comments can only appear in the middle.
+pub struct EntryWorker<'a> {
```

Comment:
what is an entry? I suggest we should choose a better name

Reply:
> what is an entry? I suggest we should choose a better name

I mean an entry of dict. This file is mainly designed for DictItem, but also used in `DestructuringItem` and `Param`. Name it `item`? Do you have a better idea?

In the beginning I want to name it just `dict.rs`, but this also cannot describe the functionality of the module well.